lib/ansible/module_utils/urls.py

```diff
@@ -1465,11 +1465,11 @@ def fetch_url(module, url, data=None, headers=None, method=None,
     except NoSSLError as e:
         distribution = get_distribution()
         if distribution is not None and distribution.lower() == 'redhat':
-            module.fail_json(msg='%s. You can also install python-ssl from EPEL' % to_native(e))
+            module.fail_json(msg='%s. You can also install python-ssl from EPEL' % to_native(e), url=to_native(url), status=None)
```

Comment:
Do I need the `to_native()` around the value of `url`?

Reply:
No, I don't think so.

Also, it might be worth just adding `status=None` to `info = dict(url=url)` and then just do `**info` in each of the `fail_json` calls.  That would avoid some unnecessary duplication.

Also, I think it is fine that this happens here.

Reply:
Awesome. I like that. I'll make that change and see about adding some unit tests.